codecov_auth/commands/owner/interactors/save_terms_agreement.py

```diff
@@ -14,6 +14,7 @@ class TermsAgreementInput:
     business_email: Optional[str] = None
     terms_agreement: bool = False
     marketing_consent: bool = False
```

Comment:
Just curious (and also a Python noob here), but on line 16 isn't the bool implied through the default assignment?
@JerrySentry @rohitvinnakota-codecov

Reply:
Got it, that totally makes sense. Only follow up I have here is do we have a stance between when we're looking to add explicit typing vs. not?

Reply:
I would say rule of thumb is always try and add types to your fn params + return values.

I personally like to type my variables if I'm using a class, like with Person above, and create dataclasses in scenarios where I need to create a helper dictionary instead

Reply:
Sounds good! Will keep that in mind.